src/components/ModelsTable.js

```diff
@@ -40,6 +45,16 @@ export default function ModelsTable({
     togglePatientIDs();
   };
 
+  // Handle download test metrics values
+  const handleDownloadTestMetricsValues = async (modelID) => {
+    let { filename, content } = await Backend.downloadTestMetricsValues(
+      keycloak.token,
+      modelID
+    );
+
+    fileDownload(content, filename);
```

Comment:
I think the `saveAs` function from the `file-saver` library could be reused here, as used in the `handleExportCSV` function, to avoid installing another similar library.